Developers/Newcomers_Guide.rst

```diff
@@ -37,9 +39,9 @@ Step 1. Meet the Community!
 ---------------------------
 
 To get started, the first step is to meet the community. We use gitter to
-communicate, and there the helpful community will guide you.
+communicate, and there, the helpful community will guide you.
```

Comment:
This comma doesnt help, from a grammar perspective.
"there" isnt helpful, especially for someone who doesnt have strong English, as this paragraph has only previously mentioned "gitter", which is a tool, not a place.